src/Microsoft.TestPlatform.Common/Filtering/FastFilter.cs

```diff
@@ -150,6 +150,11 @@ internal void AddOperator(Operator @operator)
                     {
                         operatorEncountered = true;
```

Comment:
Might be unrelated to this change but should we be putting out a meaningful message when we throw ArgumentException at Ln44 above?

Reply:
+1. Also might want to make `ToFastFilter` more robust by catching the `ArgumentException`.